west.yml

```diff
@@ -51,7 +51,7 @@ manifest:
       revision: 0ec40aed8087f26bd9ac1b70fb5a6c326a6451aa
       path: modules/hal/st
     - name: hal_stm32
-      revision: 8066e1d2477860a106e3c2e6ca841b4abbfbaad2
+      revision: pull/16/head
```

Comment:
Shouldn't this be a08f60768042be9b017796e13c5bb7f977d51a92?

You're stating that you want to use a PR as the HEAD for the STM32 HAL code.

Reply:
@psidhu , yeah, I'm actually following the process documented here: https://docs.zephyrproject.org/latest/guides/modules.html?highlight=modules#process
Now that CI has been run successfully, `pull/16/head` has been merged and  I can change to SHA1 of the merged commit.